server/src/resolvers/args/MemberArgs.ts

```diff
@@ -35,38 +35,33 @@ export class UpdateUsernameArgs {
 	@Field()
 	@MinLength(3, { message: ErrorMessages.USERNAME_MUST_BE_LONG_ERROR_MESSAGE })
 	username: string;
-
-	@Field()
-	id: string;
 }
 
 @ArgsType()
 export class UpdateEmailArgs {
 	@Field()
 	@IsEmail()
 	email: string;
-
-	@Field()
-	id: string;
 }
 
 @ArgsType()
 export class UpdatePasswordArgs {
 	@Field()
 	@Matches(passwordRegExp, { message: ErrorMessages.PASSWORD_FORMAT_ERROR_MESSAGE })
-	password: string;
+	new_password: string;
 
 	@Field()
-	@IsEmail()
-	email: string;
+	@Matches(passwordRegExp, { message: ErrorMessages.PASSWORD_FORMAT_ERROR_MESSAGE })
+	confirm_password: string;
```

Comment:
why snake_case ?

Reply:
No reason, its my brain trying code in python xDDD I will update it to our reality.